### PR TITLE
[MODULAR] [NO GBP] Quick hotfix - fixes vox spawning without their internals opened

### PR DESCRIPTION
**Pull request by vinylspiders**
## About The Pull Request

This is just a quick fix to vox outfits for something that I forgot in #18614--this will make it so they are able to benefit from #19054 (auto opening internals in the equip proc).

## How This Contributes To The Skyrat Roleplay Experience

No more vox spawning with internals toggled off.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
 
image

</details>

## Changelog

:cl:
fix: hotfixes vox spawning with their internals closed
/:cl:

### CODE REVIEW

**Comment by vinylspiders** [comment]
Whoops-- sorry @tf-4 , forgot to add this to #18614, I was waiting until after #19054 was merged but never got around to it.

**Review by tf-4 — APPROVED** [review]

**Pull request merged**
